lib/index.js

```diff
@@ -25,92 +6,95 @@ const Schema = require('./schema');
 const Models = require('./models');
 const DB = require('./DB');
 const Pkg = require('../package.json');
+const instances = require('./instances');
 
-// Module globals
-const internals = {};
+module.exports = {
```

Comment:
Refactoring the codebase to make it more readable is a nice thought and is welcomed but IMO it would be better done as a separate PR because otherwise it just adds noises to your PR and make it harder to review the real changes introduced by the new feature. AFAICT the refactoring is not needed for your feature to work, is it?

Reply:
Yes, it works without the refactoring. I refactored it to improve the readability and for understand the library to insert my proposal. I didn't think about doing this into another PR. I can send it into another PR if you prefer.